docs/current_docs/configuration/modules.mdx

```diff
@@ -150,3 +150,19 @@ When a package manager is not explicitly defined within the `package.json` file,
 :::warning
 This behavior however should be considered a sensible fallback, and not as an explicit configuration. Since this default can change, we encourage you to configure a package manager explicitly.
 :::
+
+### Alternative base images
```

Comment:
@TomChv what is the relation between this option and the `dagger.runtime` option documented above in this same page?

Reply:
`dagger.runtime` sets the js runtime - like `node` or `bun` or maybe one day `deno`.

The image to use is *derived* from the version in the `dagger.runtime` if it's present - `baseImage` allows specifying the exact image.

Reply:
Added